### PR TITLE
Feature/svr 1450 increase official group member limit

### PR DESCRIPTION
**Pull request by wolf-mik**

### CODE REVIEW

File: config/test.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          source: group
+          destination: group.delete
+          bindingKey: group.*.operation.delete
+        groupCreate:
+          source: group
+          destination: group.create
+          bindingKey: group.*.operation.create
+  defaults:
+    subscriptions:
+      deferCloseChannel: 0
```
Comment (mjbrook) [inline config/test.yaml:53]: I don't think any of these configurations are required?

File: package.json [hunk 2]
```diff
@@ -1,50 +1,60 @@
...
-    "nock": "^12.0.3",
-    "supertest": "^4.0.2"
+    "ioredis": "4.2.0",
+    "mocha": "^7.2.0",
+    "mysql2": "^1.7.0",
+    "nock": "^13.1.3",
+    "should": "^7.1.0",
+    "sinon": "^8.1.1",
+    "supertest": "^3.3.0",
+    "uuid": "^3.4.0"
```
Comment (mjbrook) [inline package.json:47]: Can we tidy up these dependencies? Many are not required.

**Comment by mjbrook** [comment]
In general, there is a lot of stuff that you've copied over from the service template that is redundant here. Things like tables and setup that needs to be tidied up.

File: package.json [hunk 3]
```diff
@@ -1,50 +1,60 @@
 {
   "name": "service-template",
-  "version": "0.0.1",
-  "description": "Template for our REST services using Nodejs, Swagger and Express",
+  "version": "1.0.0",
+  "description": "REST service for Group CRUD operations",
```
[the remark this answers is not in the document]
Comment (wolf-mik) [reply package.json:4]: I will update it

File: scripts/group-generator/resources/data.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+// Variables
+const dbSetupCfg = config.get('dbSetup');
+const dbPool = new DbPool(dbSetupCfg);
+
+const generateGroupSubscribers = async(groupSize, groupName) => await dbPool.executeQuery(sqlResource.dropGenerateGroupSubscribersProcedure)
+    .then(async() => await dbPool.executeQuery(sqlResource.createGenerateGroupSubscribersProcedure))
+    .then(async() => await dbPool.executeQuery(sqlResource.callGenerateGroupSubscribersProcedure, [groupSize, groupName]))
+    .then(async() => (await dbPool.executeQuery(sqlResource.selectGroupId))[0].groupId)
+    .then(async(groupId) => {
```
Comment (mjbrook) [inline scripts/group-generator/resources/data.js:19]: The `then` usage here is redundant when the inner function is simply awaiting an async function.

**Review by mjbrook — APPROVED** [review]

**Pull request merged**